src/coap_gnutls.c

```diff
@@ -819,7 +827,7 @@ post_client_hello_gnutls_pki(gnutls_session_t g_session)
       if ((ret = setup_pki_credentials(
                            &g_context->sni_entry_list[i].pki_credentials,
                            g_context,
-                           &sni_setup_data)) < 0) {
+                           &sni_setup_data, 0)) < 0) {
```

Comment:
Maybe we can define two pre-processor symbols for this flag to make the semantics more explicit?

Reply:
Code changes with this concept done across all the DTLS interface modules